src/include/cpu/power/spr.h

```diff
@@ -3,23 +3,23 @@
 
 #include <arch/byteorder.h>	// PPC_BIT()
 
-#define SPR_DEC					0x16
+#define SPR_DEC					22
 #define SPR_DEC_IMPLEMENTED_BITS		56
 #define SPR_DEC_LONGEST_TIME			((1ull << (SPR_DEC_IMPLEMENTED_BITS - 1)) - 1)
 
-#define SPR_SRR0				0x2A
-#define SPR_SRR1				0x2B
+#define SPR_SRR0				26
+#define SPR_SRR1				27
```

Comment:
```suggestion
#define SPR_SRR0				42
#define SPR_SRR1				43
```

Reply:
That's the issue, they were wrong previously (my bad). 26 and 27 are correct values: https://wiki.raptorcs.com/w/images/c/cb/PowerISA_public.v3.0B.pdf page 971